handlers/new-product-api/src/main/scala/com/gu/newproduct/api/addsubscription/AddDigipackSub.scala

```diff
@@ -49,15 +49,32 @@ class AddDigipackSub(
     zuoraRatePlanId <- getZuoraRateplanId(request.planId)
       .toApiGatewayContinueProcessing(internalServerError(s"no Zuora id for ${request.planId}!"))
       .toAsync
+    ratePlans = request.discountRatePlanId
+      .map(id =>
+        List(
+          ZuoraCreateSubRequestRatePlan(
+            productRatePlanId = id,
+            maybeChargeOverride = None,
+          ),
+          ZuoraCreateSubRequestRatePlan(
+            maybeChargeOverride = None,
+            productRatePlanId = zuoraRatePlanId,
+          ),
+        ),
+      )
+      .getOrElse(
+        List(
+          ZuoraCreateSubRequestRatePlan(
+            maybeChargeOverride = None,
+            productRatePlanId = zuoraRatePlanId,
+          ),
+        ),
+      )
```

Comment:
```suggestion
        ratePlans = List(
      ZuoraCreateSubRequestRatePlan(
        maybeChargeOverride = None,
        productRatePlanId = zuoraRatePlanId,
      ),
    ) ++ request.discountRatePlanId
      .map(id =>
        List(
          ZuoraCreateSubRequestRatePlan(
            productRatePlanId = id,
            maybeChargeOverride = None,
          ),
        ),
      )
      .getOrElse(Nil)
```

I think this is actually better because we don't have to define the second rate plan twice